src/styles/_helpers.scss

```diff
@@ -1,3 +1,28 @@
 .rochester {
-  font-family: $rochester;
-}
\ No newline at end of file
+	font-family: $rochester;
+}
+
+#rochester-p {
+	font-family: $rochester;
+	font-size: 48px;
```

Comment:
why don't you use `rem`s

Reply:
Was strictly following the figma. Will modify.